docs/security/lcmaps-voms-authentication.md

```diff
@@ -50,7 +47,6 @@ Additionally, there is [optional configuration](#optional-configuration) if you
 To configure your host to use LCMAPS VOMS plugin authentication, edit `/etc/osg/config.d/10-misc.ini` and set the following options:
 
 ``` ini
-glexec_location = UNAVAILABLE
```

Comment:
@matyasselmeci was saying that, if glexec_location is present and uncommented, it must be set to UNAVAILABLE. So do you want to say more about it? We might even suggest doing what I ended up doing, which was to throw out my old 10-misc.ini and start over with the .rpmnew file. Anyway, think carefully about likely or common current 10-misc.ini files and what sort of advice we want to give in the various cases.